performance/tests/OverallView.test.js

```diff
@@ -0,0 +1,186 @@
+const TestResults = require('../TestResults');
+const {CLS, FID, FPS, getAverageFPS, PageLoadingMetrics} = require('../TraceModel');
+const {clsValue, firstInputValue, getFileName} = require('../utils');
+
+describe('OverallView', () => {
+	const component = 'Overall';
+	TestResults.newFile(component);
+
+	it('FPS', async () => {
+		await FPS();
+		await page.goto('http://localhost:8080/overallView');
+		await page.waitForSelector('#virtualGridList');
+		await page.keyboard.down('ArrowUp');
+		await page.keyboard.up('ArrowUp');
+		await page.waitForTimeout(200);
+		await page.keyboard.down('ArrowUp');
+		await page.keyboard.up('ArrowUp');
+		await page.waitForTimeout(200);
+		await page.keyboard.down('ArrowUp');
+		await page.keyboard.up('ArrowUp');
```

Comment:
I don't understand what is the process of this.
Are you constantly pressing the up button on the "Display" tab button?
Could you please leave more code description about the test scenario

Reply:
belows are the same.

Reply:
I wanted to focus and tigger value changes on various components from each panel. I think it is relevant for calculating an average FPS

Reply:
I think the up buttons does nothing because initial focus is on the "Display" tab button. (there is no place to move the focus)